ipa325_wsg50/src/WSG50ROSNode.cpp

```diff
@@ -44,6 +41,7 @@
 
 // Gripper Default Values
 //
+#define MAXWIDTH 110.0 
```

Comment:
why do we need the maxwidth here and will this collide with a 210/68 gripper size?